cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,113 @@
+    </ul>
+
+<% var license = licenseInfo[model.type]; %>
+<% if(license && !_.isEmpty(license.options)) { %>
+    <div class="wrapper-license-options">
+        <h4 class="label setting-label">
+            <%- gettext("Options for {license_name}").replace("{license_name}", license.name) %>
+        </h4>
+        <p class='tip tip-inline'>
+            <%- gettext("The following options are available for the {license_name} license.")
+                        .replace("{license_name}", license.name) %>
+        </p>
+        <ul class="license-options">
+            <% _.each(license.option_order, function(optionKey) { %>
+                <% var optionInfo = license.options[optionKey]; %>
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                        class="action-item license-button
+                               <% if (optionSelected) { print("is-selected"); } %>
+                               <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <input type="checkbox"
+                               id="<%- model.type %>-<%- optionKey %>"
+                               name="<%- model.type %>-<%- optionKey %>"
+                               aria-describedby="<%- optionKey %>-explanation"
+                               <% if(optionSelected) { print('checked="checked"'); } %>
+                               <% if(optionDisabled) { print('disabled="disabled"'); } %>
+                        />
+                        <label for="<%- model.type %>-<%- optionKey %>" class="option-name">
+                            <%- optionInfo.name %>
+                        </label>
+                        <div id="<%- optionKey %>-explanation" class="explanation">
+                            <%- optionInfo.help %>
+                        </div>
+                    </li>
+                <% } // could implement other types here %>
+            <% }) %>
+        </ul>
+    </div>
+<% } %>
+
+<% if (showPreview) { %>
+    <div class="license-preview-wrapper">
+        <h4 class="label setting-label">
+            <%= gettext("License Display") %>
+        </h4>
+        <p class="tip">
+            <%= gettext("The following message will be displayed at the bottom of the courseware pages within your course.") %>
+        </p>
+        <div class="license-preview">
+        <% // keep this synchronized with the contents of common/templates/license.html %>
+        <% if (model.type === "all-rights-reserved") { %>
+            © <span class="license-text"><%= gettext("All Rights Reserved") %></span>
```

Comment:
Did you reach a conclusion for how to include copyright symbols? Should we use ``&copy;``? How does it need to be handled for i18n and right-to-left?